### PR TITLE
modify Jenkinsfile + Dockerfile for `terraform-provider-rancher2/tfp-automation` tests to run in our Jenkins env

### PR DESCRIPTION
**Pull request by Josh-Diamond**
integrate `rancher/terraform-provider-rancher2/tfp-automation` tests into Jenkins env, as part of https://github.com/rancher/qa-tasks/issues/386

### CODE REVIEW

**Review by markusewalker — APPROVED** [review]
LGTM

File: tfp-automation/Jenkinsfile [hunk 1]
```diff
@@ -1,26 +1,29 @@
-pipeline {
-  agent any
-
-  stages {
+node {
```
Comment (slickwarren) [inline tfp-automation/Jenkinsfile:2]: I don't think things break if you omit this, but all our other jenkinsfiles have the shebang `#!groovy` as the first line of each file.

**Review by slickwarren — COMMENTED** [review]
small nit, everything else lgtm

File: tfp-automation/Jenkinsfile (hunk 1 above)
Comment (Josh-Diamond) [reply tfp-automation/Jenkinsfile:2]: I had that initially, and it threw an error so i took it out, i'll add it back rn and re-test this and will DM you results 👍

Comment (Josh-Diamond) [reply tfp-automation/Jenkinsfile:2]: not hitting the error today, so likely encountered that error yesterday due to some syntax issue, but not reproducing it now, so  I'll update PR to include `#!groovy` 👍

**Review by slickwarren — APPROVED** [review]

**Pull request merged**